### ISSUE TITLE
[20220612] Weekly AI ArXiv 만담 - 54회차

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#54]
### News
- Conferences
  - CVPR 2022: 6.19 ~ 24 (New Orleans)
- 대기업 중심의 AI 투자 관련 (SK, LG, KT 등등)
- [스캐터랩, 정부와 AI 윤리점검표 개발 추진](https://n.news.naver.com/mnews/article/092/0002259047?sid=105)

### ArXiv
- [Revisiting the "Video" in Video-Language Understanding](https://arxiv.org/abs/2206.01720)
  - Video-language benchmark 들이 정말 video (temporal) 이해를 평가하는 것이 맞는가? 에 대한 고찰 연구(from Stanford U. 와 Toyota Research)
  - Frozen retrained image-text model을 활용한 Atemporal probe 모델을 제안해서 temporal 정보를 배제하여 가장 유용한 single frame만으로 video-language downstream을 풀어봄 --> 생각보다 아주 잘됨
  - ATP가 제대로 못풀어야 causal & tempral 정보가 필요한 video-language 벤치마크가 될 것
![image](https://user-images.githubusercontent.com/11782739/173193573-90d1b876-b5bb-4533-a509-8fb277d8886a.png)
![image](https://user-images.githubusercontent.com/11782739/173193589-61d090c6-75bf-4204-94f3-adb929074546.png)

- [On Data Scaling in Masked Image Modeling](https://arxiv.org/abs/2206.04664)
  - Masked Image Modeling 형태의 self-supervised learning에서 데이터 크기 + 학습 epoch + 모델 크기가 주는 영향 분석 (from 칭화대, MSRA)
  - MIM은 SimMIM, 모델은 Swin-V2의 여러가지 크기로 함
  - 학습데이터는 ImageNet-1k와 21k를 활용, 1k중 학습데이터에 활용 비율 조정. 
  - 결론은 예상가능한 내용
     - 데이터 많이 쓰면 성능 좋아짐, 모델 큰데 데이터 작으면 overfit, 
     - 데이터가 많을 때는 학습오래하면 유리함
     - validation loss는 downstream task 성능 예측에 좋은 지표가됨
![image](https://user-images.githubusercontent.com/11782739/173194524-64404e08-afe2-418e-8c88-b3831bf67beb.png)
![image](https://user-images.githubusercontent.com/11782739/173194551-91a3018a-74e4-497f-9c60-75df18a19628.png)

- 흥미있는 연구
  - [Beyond the Imitation Game: Quantifying and extrapolating the capabilities of language models](https://arxiv.org/abs/2206.04615)
    - PaLM에서도 사용된 언어모델 평가용 BIG-Bench 데이터셋 (132개 기관 442명)
  - [Extreme Masking for Learning Instance and Distributed Visual Representations](https://arxiv.org/abs/2206.04667)
    - MSRA에서 나온.. MIM ViT에서 masked비율을 75%보다 훨씬 키워 학습?
  - [VideoINR: Learning Video Implicit Neural Representation for Continuous Space-Time Super-Resolution](https://arxiv.org/abs/2206.04647)
    - INR로 In-the-wild video super resolution을 (CVPR2022 oral)

**Comment by ghlee0304** [comment]
- [StyleTTS: A Style-Based Generative Model for Natural and Diverse Text-to-Speech Synthesis](https://arxiv.org/pdf/2205.15439.pdf)
- [Guided-TTS 2: A Diffusion Model for High-quality Adaptive Text-to-Speech with Untranscribed Data](https://arxiv.org/pdf/2205.15370.pdf)
- [AdaVITS: Tiny VITS for Low Computing Resource Speaker Adaptation](https://arxiv.org/pdf/2206.00208.pdf)
- [Dict-TTS: Learning to Pronounce with Prior Dictionary Knowledge for Text-to-Speech](https://arxiv.org/pdf/2206.02147.pdf)
- [Zero-Shot Voice Conditioning for Denoising Diffusion TTS Models](https://arxiv.org/pdf/2206.02246.pdf)
- [UTTS: Unsupervised TTS with Conditional Disentangled Sequential Variational Auto-encoder](https://arxiv.org/pdf/2206.02512.pdf)
- [BigVGAN: A Universal Neural Vocoder with Large-Scale Training](https://arxiv.org/pdf/2206.04658.pdf)

**Comment by Clyde21c** [comment]
[Multi-Game Decision Transformers](https://arxiv.org/abs/2205.15241)

**Comment by veritas9872** [comment]
**AI's New Frontier**

![Screenshot (79)](https://user-images.githubusercontent.com/33523965/173233025-5152c65d-5175-4328-ab78-0f83bf0d65cb.png)


영국 Economist지에서 인공지능과 Foundational Model에 대한 특집을 이번주에 냈습니다.

기술전문 신문이 아님에도 불구하고 최근에 나온 GATO 모델 등에 대해서 매우 자세하고 논의하고 일반인의 입장에서도 Foundation model 및 자연어 모델, DALL-E 등 초거대 모델과 이로 인해 발생하는 이슈 등에 대해 매우 잘 정리했기 때문에 비전문가에게 분야 및 이슈를 소개하는데 매우 좋은 기사라고 생각됩니다.

![Screenshot (77)](https://user-images.githubusercontent.com/33523965/173233036-6cca33d6-256b-4ed8-bcb5-5be22c96043f.png)
![Screenshot (78)](https://user-images.githubusercontent.com/33523965/173233043-2d703bfe-a579-4adc-8522-e87116c861ff.png)


또한, 저명한 인지과학자이신 Douglas Hofstadter의 GPT3와의 adversarial한 대화를 통해 아직까지도 인공지능이 겉으로는 세계에 대해 이해를 하고 있는 것으로 보일 수 있지만 진정한 World Model을 가지지 못하는 것을 보이는데 최근 이슈가 된 AI 모델과 의식의 관계에 대해 이미 많이 고민을 해본 사람에게도 새로운 각도를 제시해주어 도움이 될 수 있다고 생각합니다.

![Screenshot (76)](https://user-images.githubusercontent.com/33523965/173233067-1d249b96-5510-4636-95e8-3d0b10bdc882.png)


참조를 위해 이번 주 기사 링크를 공유해드립니다.

https://www.economist.com/interactive/briefing/2022/06/11/huge-foundation-models-are-turbo-charging-ai-progress

https://www.economist.com/by-invitation/2022/06/09/artificial-neural-networks-today-are-not-conscious-according-to-douglas-hofstadter

**Comment by veritas9872** [comment]
**An on-chip photonic deep neural network for image classification**
URL: https://www.nature.com/articles/s41586-022-04714-0
Paper: https://arxiv.org/ftp/arxiv/papers/2106/2106.11747.pdf

**Quantum advantage in learning from experiments**
URL: https://www.science.org/doi/10.1126/science.abn7293
Paper: https://arxiv.org/pdf/2112.00778.pdf

딥러닝에 양자 컴퓨터를 사용하는 논문이 각각 Nature와 Science에 accept되었습니다.
초대형 딥러닝 모델이 이슈화되면서 1조 parameter 이상의 모델은 기존의 컴퓨터에서 실행하는데 data transfer로 인해 실행이 느려질 수 밖에 없는데 레이저를 이용할 경우 모델 구조마다 전용 프로세서를 만들어야한다는 단점이 있지만 대신 광속의 실행 시간을 적용할 수 있다는 매우 큰 장점을 가지기 때문에 초거대 신경망을 사용하는데 도움이 될 수 있다고 생각됩니다. 특히 최근 모델들은 대체로 비슷한 구조를 중첩한 구조이기 때문에 더더욱 도움이 될 것으로 생각됩니다.

**Beyond the Imitation Game: Collaborative benchmark for measuring and extrapolating the capabilities of language models**
Paper: https://arxiv.org/pdf/2206.04615.pdf
GitHub: https://github.com/google/BIG-bench
![Screenshot (82)](https://user-images.githubusercontent.com/33523965/173233688-eed01f65-a60a-4b6d-bb79-297073bdb189.png)
![Screenshot (83)](https://user-images.githubusercontent.com/33523965/173233692-345ad116-ab48-462b-9cea-4bf235b2465d.png)

**Comment by hollobit** [comment]
### IBM Global AI Adoption Index 2022
https://www.ibm.com/watson/resources/ai-adoption
https://www.ibm.com/downloads/cas/GVAGA3JP
>- 2022년 세계 기업의 평균 AI 도입률은 34%로 전년 대비 13% 
>- 국내 기업 AI 도입 격차...1000명 이상 93% vs 1000명 이하 57%
>- 국내 기업은 AI를 도입하는 주요 목적으로 IT운영 효율성 제고(40%), 비즈니스 프로세스 효율성 제고(39%), 비용 절감(39%)
>- 전세계적으로 기업은 설명 가능하고 신뢰할 수 있는 AI 개발과 관련하여 여러 장벽에 직면
>   - 63% Lack of skills and training to develop and manage trustworthy AI
>   - 60% AI governance and management tools that don’t work across all data environments
>   - 59% Lack of an AI strategy
>   - 57% AI outcomes that aren’t explainable
>   - 57% Lack of company guidelines for developing trustworthy, ethical AI
>   - 57% AI vendors who don’t include explainability features
>   - 56% Lack of regulatory guidance from governments or industry
>   - 56% Building models on data that has inherent bias (social, economic, and so on)
>- (AI 도입을 저해하는 요소) ‘AI 기술, 지식 및 전문성 부족(45%)’, ‘AI 모델 개발을 위한 도구 및 플랫폼 부족(39%)’, ‘지나치게 높은 가격(33%)’

**Comment by kwonminki** [comment]
### Elucidating the Design Space of Diffusion-Based Generative Models  | [arxiv](https://arxiv.org/abs/2206.00364)
![image](https://user-images.githubusercontent.com/33779055/173233950-544f1a3f-b3d2-439f-beec-2bf3ca791f30.png)
![image](https://user-images.githubusercontent.com/33779055/173234034-d482b6c7-bda2-49ff-bbf9-fce533a1cc19.png)
Tero Karras 1저자.. 이젠 DIffusion까지 건드는건가 싶네요.  

정말 케라스 스타일 그대로 많은 실험과 뛰어난 직감을 가지고 모델을 재구성한 듯 합니다.

- 하이퍼 파라미터 조절 (dependent 하도록)
- 모델의 약간의 변화 
-  σ-dependent normalization factor 의 수정. (기존에는 작을 땐 너무 작고 클 땐 너무 컸음)
- σ(time step t)에 따른 loss 변화
- data agumentation
